biospecimen-tracking/api/src/main/java/org/phenotips/data/internal/controller/Biospecimen.java

```diff
@@ -24,14 +24,13 @@
 import java.util.Set;
 
 import org.apache.commons.collections4.CollectionUtils;
-import org.apache.commons.configuration.plist.ParseException;
 import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTime;
 import org.joda.time.format.DateTimeFormatter;
 import org.joda.time.format.ISODateTimeFormat;
-import org.json.JSONException;
 import org.json.JSONObject;
 
+import com.sun.star.auth.InvalidArgumentException;
```

Comment:
I think you're looking for [IllegalArgumentException](https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalArgumentException.html).